app/lib/ui/about/about_screen.dart

```diff
@@ -35,27 +32,25 @@ class _AboutScreenState extends State<AboutScreen> {
   Widget build(BuildContext context) {
     return Scaffold(
         body: SafeArea(
+            child: SingleChildScrollView(
+      child: Column(
+        children: [
+          Image(image: AssetImage('assets/il_about.png')),
```

Comment:
I think we can use these lines to allow the image to take all the space available when device is in landscape:
```dart
          Image(
            width: double.infinity,
            image: AssetImage('assets/il_about.png'),
            fit: BoxFit.cover,
          ),
```